events/message.js

```diff
@@ -70,6 +70,12 @@ module.exports = class {
     
     this.client.logger.log(`${this.client.config.permLevels.find(l => l.level === level).name} ${message.author.username} (${message.author.id}) ran command ${cmd.help.name}`, "cmd");
     
+    if (message.channel.type === "text") {      
```

Comment:
This channel type check isn't required at all as Misaki actively ignores any channel that doesn't have a guild object.

Reply:
She doesn't respond in DMs?

Reply:
[What do you think?](https://github.com/NotAWeebDev/Misaki/blob/master/events/message.js#L9)